scipy/special/_add_newdocs.py

```diff
@@ -5535,6 +5535,75 @@ def add_newdoc(name, doc):
     gdtria : Inverse with respect to `a` of `gdtr(a, b, x)`.
     gdtrix : Inverse with respect to `x` of `gdtr(a, b, x)`.
 
+    Notes
+    -----
+
+    The cumulative distribution function `p` is computed using the Cephes [1]_
+    routines `igam` and `igamc`. Computation of `b` involves a search for a value
+    that produces the desired value of `p` using a bracketing root finding method
+    developed by Bus & Dekker [2]_.
```

Comment:
Guess the reference needs to be updated now that you’ve switched to chandrupatla solver.